tests/Testdata/pos1.json

```
@@ -0,0 +1,11 @@
+{
+	"fullWidth": 13934,
```

Please indent the lines evenly.

```suggestion
  "fullWidth": 13934,
```